change/@ni-nimble-components-f58de85c-8a74-4a0b-b1c4-fdab36bfa94f.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

Comment:
Opinions on calling this a `major` change in case people were relying on these slots being visible? I think I'm ok with `patch` since the APIs were undocumented (incubating?) but could also see a client being annoyed if they were using it and we didn't provide any notification that it was going away.

My vote is to keep it `patch` but post in the UI WG and Web Chapter Blazor channels to give clients a heads up when this goes in.

Reply:
+1 for patch, it's an internal api to use those slots and not a public api so not breaking 🙃.
Sure not well documented or anything. Should read our minds better 🔮.

Reply:
I went back and forth on this myself, and decided on patch since it should really only help applications better align with nimble styling (not actually break an application or build). Whether or not that is actually true, I'm not sure. I'll leave this as patch since that's the way we are all leaning.

Reply:
I'll leave this conversation open as a reminder to post in the UI WG and Web Chapter Blazor channels when this change goes in.